### ISSUE TITLE
Ability to select Default call number

### ISSUE THREAD

**Issue by coderGtm** [coderGtm/yantra-app-launcher#103]
A flag or something to select a default number when multiple numbers are available for a contact name

**Comment by disco-volante** [comment]
...the same for applications.

**Comment by coderGtm** [comment]
Yeah both will be done. Till then you can make aliases to open apps quickly with their package names
